### DISCUSSION TITLE
Landscape view.

### DISCUSSION THREAD

**Discussion by ddmmcz** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#274] [Show and tell]
Hi, thank you for a great job! I would like to ask if it is possible to move the control part vertically of the map, I can see the display on the tablet wide. Is there such a possibility?

**Comment by PiotrMachowski** [comment]
See this issue: #183

**Reply by ddmmcz** [reply]
I'm sorry I didn't notice, thank you very much for your reply!
